Rubberduck.CodeAnalysis/Inspections/Concrete/UnreachableCaseEvaluation/UnreachableCaseInspector.cs

```diff
@@ -98,23 +98,26 @@ internal class UnreachableCaseInspector : IUnreachableCaseInspector
 
         private UnreachableCaseInspection.CaseInspectionResultType? InvalidRangeExpressionsType(ICollection<IRangeClauseExpression> rangeClauseExpressions)
         {
-            if (rangeClauseExpressions.Any(expr => expr.IsMismatch))
+            if (rangeClauseExpressions.Any(expr => expr?.IsMismatch ?? false))
```

Comment:
1) why is this in this PR? I don't see this mentioned in the main comments?
2) Instead of sprinkling null operators all over, why not just `if (expr == null) return null` ?

Reply:
Ah, they're edit-and-continue tweaks introduced as a result of hitting a NRE there - indeed it would be cleaner to have a proper guard clause... it doesn't quite belong in this PR but I'll clean this up rather than revert it.